### DISCUSSION TITLE
Please help me to configure PINNIPED _USERNAME & PINNIPED_PASSWD from Linux server

### DISCUSSION THREAD

**Discussion by shivkumarpatil** [vmware-tanzu/pinniped#1254] [Q&A]
Hi All,

I started working on Tanzu kubernetes cluster. My client aksed me to connect Kubernetes cluster from Linux server by using Ansible playbook.

Please help me in this. Thank you.

My mail id: shivva45@gmail.com

**Comment by anjaltelang** [comment]
@shivkumarpatil A good place to start will be this blog https://pinniped.dev/posts/supporting-remote-oidc-workflows/ which has a demo video and steps you can use to configure workflow from a jump host. Please also feel free to join and ask in the Kubernetes Pinniped channel (https://go.pinniped.dev/community/slack)
